Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.9
+
+ADD requirements.txt /tmp/requirements.txt
+
+RUN pip install -r /tmp/requirements.txt
+
+ADD src /var/server/src
+ADD assets /var/server/assets
```

Comment:
Just to make sure, do we need the data folder as well?

Reply:
For running the inference server alone we don't, we only need the `models` directory. It depends on if we want to run something like `dvc repro` in the image. I'm not sure how we want to do it so I just added everything.